twake/frontend/src/app/deprecated/workspaces/workspaces.js

```diff
@@ -158,6 +159,9 @@ class Workspaces extends Observable {
     }
 
     LocalStorage.setItem(`default_workspace_id_${workspace.company_id}`, workspace.id);
+    UserAPIClient.setUserPreferences({
```

Comment:
Could you move this into channels-bar/ChannelsBar.tsx with a useSetLastWorkspacePreference();
The hook will then look like this:
```
const workspaceId = useRouterWorkspace();
useEffect(()=>{
 UserAPIClient.setUserPreferences({...});
}, [workspaceId])
```

Reply:
Done here in `twake/frontend/src/app/features/users/hooks/use-set-last-workspace-preferences.ts`